### ISSUE TITLE
Archived GitLab projects are added to the job queue

### ISSUE THREAD

**Issue by TobiX** [mend/renovate-ce-ee#461]
I know that renovate bails early for archived projects, but it would be even nicer if those projects aren't discovered at all. We have quit a lot of archived projects and this creates a lot of "useless" runs (and shows a "wrong" number in the `repositoryCount` of the admin API). 

```
2024-02-20T14:33:21.233 renovate[6491] DEBUG    Repository is archived - throwing error to abort renovation
```

If you are using the GitLab projects REST API (https://docs.gitlab.com/ee/api/projects.html#list-all-projects), it's probably enough to add `archived=false` to the URL.

**Comment by justo-mend** [comment]
Good suggestion. Thanks for raising it. We will action this.

**Comment by TobiX** [comment]
This seems fixed with 8.2.0? `repositoryCount` in the API dropped significantly.

**Comment by PhilipAbed** [comment]
@TobiX yes.
